src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,42 @@
 package core.basesyntax;
 
+abstract class Machine {
```

Comment:
You should create class Machine, separate class Truck etc. 
Don't create classes into one